src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="APPLE A1419 iMac"
+      />
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__raiting">
+        <div class="stars stars--4">
+          <div class="star"></div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
```
write it following the BEM